lib/LanguageServerNameImport/Service/NameImport.php

```diff
@@ -0,0 +1,90 @@
+<?php
```

Comment:
@dantleech I extracted the import functionality to this "service". Could you please have a look whether you are fine with the naming and location of it? Thanks :)

(Using this new service is still open. See ToDo in the PR description.)

Reply:
Could put it in `Model` I guess (it's both code-transform and language-server so nothing "extrenal" really)